src/main/resources/templates/index.html

```diff
@@ -1,13 +1,20 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org/">
 
-<head th:replace="fragments :: head">
+<head >
+    <div th:replace="fragments :: head" th:remove="tag">
+    </div>
+    <style>
+        .home-link {
+            display: none;
+        }
+    </style>
     <meta charset="UTF-8"/>
     <title>Kansas City Water Garden Tours</title>
 </head>
 
-
 <body>
+<div class="home" th:replace = "fragments :: page-header"></div>
```

Comment:
I don't think this line is needed, since this class isn't referenced anywhere else. This line and line 19 are making two navbars show up on index.html

Reply:
I saw that, too.  Thanks for the reminder, I think it’s fixed now.